tests/tensorizer_loader/test_tensorizer.py

```diff
@@ -233,14 +245,18 @@ def test_openai_apiserver_with_tensorizer(vllm_runner, tmp_path):
 
 
 def test_raise_value_error_on_invalid_load_format(vllm_runner):
+    model = None
     with pytest.raises(ValueError):
-        vllm_runner(
+        model = vllm_runner(
             model_ref,
             load_format="safetensors",
             model_loader_extra_config=TensorizerConfig(tensorizer_uri="test"))
+    del model
+    gc.collect()
+    torch.cuda.empty_cache()
 
 
-@pytest.mark.skipif(torch.cuda.device_count() < 2,
+@pytest.mark.skipif(torch.cuda.device_count() < 3,
```

Comment:
why change it from 2 to 3?

Reply:
This was an internal testing change; I reverted it.